features/cli/oc_import_image.feature

```diff
@@ -83,13 +83,15 @@ Feature: oc import-image related feature
   Scenario: OCP-12765:ImageRegistry Allow imagestream request deployment config triggers by different mode('TagreferencePolicy':source/local)
     Given I have a project
     When I run the :tag client command with:
+      | import_mode | PreserveOriginal                                      |
```

Comment:
Do we support this option in old versions. e.g, 4.10 ?

Reply:
@aleskandro do we support it from 4.13?

Reply:
This is supported for 4.13+. For cucushift we need something to add the parameter starting from that version, if possible

Reply:
@liangxia so we need to add a version check for cucushift test?